Makefile

```diff
@@ -23,6 +23,8 @@ ABS_ROOT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))/
 # This is used to include garbage in the signing process to test verification
 # errors in the integration test. It should not be used for any real builds.
 TEST_EXTRA_SIGN ?= /dev/null
+# Final image size in MiB
+ROM_SIZE := 16
```

Comment:
Could you make it so there is one non-padded artifact and one that is padded? I'm concerned about losing track of the image size and letting it grow over time otherwise.

Reply:
Mhh, with each call of the target the image gets replaced by an empty file (without notrunc), so that shouldn't happen but maybe I don't get what you mean

Reply:
bluecmd ping

Reply:
Thanks for the ping. What I mean is if u-bmc compiles to 10 MB I would like flash.img to be 10 MB. That way it's up to the user how they wishes to pad the image. Right now you add a requirement that the image be at least 16 MB, which is a regression in my mind.

What about:

- Keeping the current flash.img but renaming it u-bmc.img.
- Make flash.img take ROM_SIZE, but warn if the output became larger, and make ROM_SIZE user settable (?= 16).

That way when we fiddle with the compression and things with u-bmc we have a build output that we can look at, while also make it easy for users to flash their image, and customize the output image to whatever size they want.

Reply:
Ah, yes, now I got it. Well, I wanted this to be user-settable (make ROM_SIZE=32 for example) but I used `:=` instead of `?=` - will be fixed.

`u-bmc.img` and `flash.img` sound good, will add that

Reply:
done